service/storage_service.cc

```diff
@@ -4661,8 +4661,15 @@ future<> storage_service::raft_check_and_repair_cdc_streams() {
         }
 
         curr_gen = _topology_state_machine._topology.current_cdc_generation_id;
-
-        // FIXME: check if the current generation is optimal, don't request new one if it isn't
+        if (!curr_gen) {
+            slogger.error("check_and_repair_cdc_streams: no current CDC generation, requesting a new one.");
+        }
```

Comment:
Nit: `else` should be on the same line.
I'll fix during merge, no need to send a new version.

Reply:
Fixed.